src/framework/shortcuts/data/shortcuts-Mac.xml

```diff
@@ -833,7 +833,7 @@
     </SC>
   <SC>
     <key>fullscreen</key>
-    <seq>Ctrl+U</seq>
+    <seq>F11</seq>
```

Comment:
F11 used to mean "toggle play panel" but that's gone in MU4, and F11 is a fairly common key in Windows apps at least to toggle full screen mode, and best of all doesn't conflict with Ctrl+U to mean "toggle underline".